### PR TITLE
Upload assets article in French

### PR DESCRIPTION
**Pull request by morganen**
FR video are missing: we will merge this PR as soon as we will implement videos in the Helpcenter ;)

### CODE REVIEW

File: content/md/assets/upload-assets_fr [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+id: upload-assets
+themes: manage-your-assets, import-export-data
+title: "**Upload** assets"
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]: Translate the title also. ;)

File: content/md/assets/upload-assets_fr [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+related: work-with-assets, assets-transformation
+---
+
+
+# Aperçu
+Le chargement en masse des ressources numériques est particulièrement utile si vous voulez mettre à jour votre catalogue avec des sources externes: shooting photo, nouvelle collection, etc. 
+
+Le PIM utilise le nom du fichier pour savoir pour quelle ressource le fichier doit être utilisé et si cette ressource est localisée ou pas. 
+
+**Exemple 1 avec une ressource non-localisable**
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]:
```suggestion
**Exemple 1 avec une ressource non-localisable**  
```
You need to add two spaces at the end of your line if you want your content to actually go to the next line.

File: content/md/assets/upload-assets_fr [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Si le nom de fichier est `image_principale_S1263547.jpg`, alors le PIM va vérifier: 
+- Si la ressource qui porte le code `image_principale_S1263547` existe déja, le PIM va mettre à jour cette ressource en important le fichier image en tant que fichier de référence (et va regénérer les fichiers de variations en fonction).
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]:
```suggestion
- Si la ressource qui porte le code `image_principale_S1263547` existe déjà, le PIM va mettre à jour cette ressource en important le fichier image en tant que fichier de référence (et va regénérer les fichiers de variations corredpondants). 
```

File: content/md/assets/upload-assets_fr [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Le chargement en masse des ressources numériques est particulièrement utile si vous voulez mettre à jour votre catalogue avec des sources externes: shooting photo, nouvelle collection, etc. 
+
+Le PIM utilise le nom du fichier pour savoir pour quelle ressource le fichier doit être utilisé et si cette ressource est localisée ou pas. 
+
+**Exemple 1 avec une ressource non-localisable**
+Si le nom de fichier est `image_principale_S1263547.jpg`, alors le PIM va vérifier: 
+- Si la ressource qui porte le code `image_principale_S1263547` existe déja, le PIM va mettre à jour cette ressource en important le fichier image en tant que fichier de référence (et va regénérer les fichiers de variations en fonction). 
+- Si la ressource n’existe pas encore, le PIM va créer une nouvelle ressource portant le code `image_principale_S1263547` avec l’image en tant que fichier de référence, de sorte que les fichiers de variations puissent être générés. 
+
+**Exemple 2 avec un média localisable**
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]:
```suggestion
**Exemple 2 avec un média localisable**  
```

File: content/md/assets/upload-assets_fr [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+Si le nom de fichier est « akene-en_US.pdf », alors le PIM va vérifier: 
+- Si la ressource qui porte le code `akene` existe déjà et si elle est localisable, le PIM va mettre à jour la ressource portant le code `akene` en important le fichier `akene-en_US.pdf` en tant que fichier de référence pour la locale donnée (code de la locale à la fin du nom du fichier: ici `en_US`).
+- Si la ressource portant le code `akene` existe déjà mais qu’elle n’est pas localisable, le PIM affichera un message d’erreur en rouge. 
+- Si la ressource portant le code `akene` n’existe pas encore, le PIM va créer une nouvelle ressource localisable ayant pour code `akene` et pour fichier de référence de la locale donnée, le fichier PDF. (Code de la locale à la fin du nom du fichier: ici `en_US`)
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]:
```suggestion
- Si la ressource portant le code `akene` n’existe pas encore, le PIM va créer une nouvelle ressource localisable ayant pour code `akene` et pour fichier de référence de la locale donnée, le fichier PDF (Code de la locale à la fin du nom du fichier: ici `en_US`).
```

File: content/md/assets/upload-assets_fr [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+:::warning
+Si la locale est désactivée ou n’existe pas dans le PIM, le PIM affichera un message d’erreur en rouge. 
+:::
+
+# Comment faire?
+
+1. Allez dans le menu des ressources numériques.
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]:
```suggestion
1. Allez dans le menu `Ressources`. 
```

File: content/md/assets/upload-assets_fr [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+- Si la ressource portant le code `akene` n’existe pas encore, le PIM va créer une nouvelle ressource localisable ayant pour code `akene` et pour fichier de référence de la locale donnée, le fichier PDF. (Code de la locale à la fin du nom du fichier: ici `en_US`)
+
+:::warning
+Si la locale est désactivée ou n’existe pas dans le PIM, le PIM affichera un message d’erreur en rouge. 
+:::
+
+# Comment faire?
+
+1. Allez dans le menu des ressources numériques. 
+1. Cliquez sur le bouton `...` en haut à droite de l’écran, et sélectionnez `Charger des ressources numériques`
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]: It's missing the screenshot here.

File: content/md/assets/upload-assets_fr [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+:::warning
+Si la locale est désactivée ou n’existe pas dans le PIM, le PIM affichera un message d’erreur en rouge. 
+:::
+
+# Comment faire?
+
+1. Allez dans le menu des ressources numériques. 
+1. Cliquez sur le bouton `...` en haut à droite de l’écran, et sélectionnez `Charger des ressources numériques` 
+1. Glissez et déposez vos fichiers dans cette page ou cliquez sur `Cliquez et déposez vos ressources numériques ou cliquez ici` pour ouvrir la fenêtre de dialogue et sélectionnez les fichiers que vous voulez charger
+1. Cliquez sur le bouton `Charger les ressources numériques` pour les charger dans le PIM.
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]: It's missing the screenshot here.

File: content/md/assets/upload-assets_fr [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+1. Allez dans le menu des ressources numériques. 
+1. Cliquez sur le bouton `...` en haut à droite de l’écran, et sélectionnez `Charger des ressources numériques` 
+1. Glissez et déposez vos fichiers dans cette page ou cliquez sur `Cliquez et déposez vos ressources numériques ou cliquez ici` pour ouvrir la fenêtre de dialogue et sélectionnez les fichiers que vous voulez charger
+1. Cliquez sur le bouton `Charger les ressources numériques` pour les charger dans le PIM. 
+1. Une fois que les fichiers sont chargés, cliquez sur le bouton `Importer` pour créer les ressources numériques dans le PIM. 
+
+Le compte rendu de l’import est affiché. 
+
+::: info
+Quand des fichiers sont ajoutés à la liste, vous pouvez les retirer peu importe leur statut.
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr]:
```suggestion
Quand des fichiers sont ajoutés à la liste, vous pouvez les retirer peu importe leur statut.  
```

File: content/md/assets/upload-assets_fr (hunk 1 above)
Comment (morganen) [reply content/md/assets/upload-assets_fr]: J'ai traduit le titre mais est-ce qu'il faut aussi traduire tout le reste ?  :)
id: _upload-assets_
themes: _manage-your-assets, import-export-data_
title: "**Charger** des ressources numériques"
popular: false
ee-only: true
related: _work-with-assets, assets-transformation_

File: content/md/assets/upload-assets_fr (hunk 7 above)
Comment (morganen) [reply content/md/assets/upload-assets_fr]: I will put a video instead of a screenshot

File: content/md/assets/upload-assets_fr.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+related: work-with-assets, assets-transformation
+---
+
+
+# Aperçu
+Le chargement en masse des ressources numériques est particulièrement utile si vous voulez mettre à jour votre catalogue avec des sources externes: shooting photo, nouvelle collection, etc.
+
+Le PIM utilise le nom du fichier pour savoir pour quelle ressource le fichier doit être utilisé et si cette ressource est localisée ou pas.
+
+<<<<<<< HEAD:content/md/assets/upload-assets_fr.md
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr.md]: Small conflict pushed here.

File: content/md/assets/upload-assets_fr.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+- Si la ressource portant le code `akene` n’existe pas encore, le PIM va créer une nouvelle ressource localisable ayant pour code `akene` et pour fichier de référence de la locale donnée, le fichier PDF (Code de la locale à la fin du nom du fichier: ici `en_US`).
+>>>>>>> e8c8868e7540bbae5d618ce4b73b2eba93b35b57:content/md/assets/upload-assets_fr
+
+:::warning
+Si la locale est désactivée ou n’existe pas dans le PIM, le PIM affichera un message d’erreur en rouge.
+:::
+
+# Comment faire?
+
+<<<<<<< HEAD:content/md/assets/upload-assets_fr.md
```
Comment (NolwennP) [inline content/md/assets/upload-assets_fr.md]: Small conflict pushed here.

**Pull request closed without merging**